### PR TITLE
Update people.md

### PR DESCRIPTION
**Pull request by jzgithub1**
Added my name to contributors

### CODE REVIEW

**Review by keith-turner — APPROVED** [review]
Thanks.  We usually put the names in alpha order by first name.  I will make this change when I merge it.

File: pages/people.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ based on commits.
 | [Michael Wall](https://github.com/mjwall)                |                                     | [ET][et]   |
 | [Ravikrishna Adepu](https://github.com/adepuravikrishna) | [SAP Labs LLC](https://sap.com)     | [PT][pt]   |
 | [Sri Kailash](https://github.com/srikailash)		       | [Booking.com](https://booking.com)  | [CEST][cst]|
+| [Jeffrey Zeiberg](https://github.com/jzeiberg)           |[ASRC Federal](https://www.asrc.com) | [ET][et]   |
```
Comment (cjmctague) [inline pages/people.md:42]: Can you put your name below "Garvit Bansal" so it's alphabetical order.

**Review by cjmctague — CHANGES_REQUESTED** [review]

**Review by cjmctague — APPROVED** [review]

File: pages/people.md (hunk 1 above)
Comment (cjmctague) [reply pages/people.md:42]: Didn't see Keith's response.

**Comment by keith-turner** [comment]
Merged changes in edbc496

**Pull request closed without merging**